src/memoize.js

```diff
@@ -41,7 +41,9 @@ var last = require('./last');
 module.exports = function memoize(fn) {
     var cache = {};
     return function() {
-        if (!arguments.length) {return;}
+        if (arguments.length === 0) {
+            return fn.apply(this, arguments);
```

Comment:
I realize this could be written as `fn.call(this)` but there's value in using the same approach as in the other branch. This makes it clear that we're calculating the return value the same way in both cases; the only difference is that we don't involve the cache if given no arguments.